SCClassLibrary/Common/Streams/History.sc

```diff
@@ -325,6 +335,34 @@ History { 		// adc 2006, Birmingham; rewrite 2007.
 		^str.clumps(indices.differentiate)
 	}
 
+	*evaluateLineAt { |index| current.evaluateLineAt(index) }
```

Comment:
If you're going to abbreviate 'eval' below, you should do it here too.

Reply:
done